src/main/components/urlBar.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+
+const UrlBar = (props) => {
```

Comment:
Can we call this `RequestBar` from our whiteboard discussion? I think that better encapsulates its full functionality (URI input, method selection, and the fetch itself)